frontend/jest/featureFlags.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
       localStorage: [
         { name: "cxg-ff-auth", value: "yes" },
         { name: "cxg-ff-cc", value: "yes" },
+        { name: "cxg-ff-gs", value: "yes" },
```

Comment:
@MDunitz Corrected the values in both `main` and this PR now 🙏 Thanks so much again for catching this!!